Tests/EndToEndTests/CNTKv2Python/Tutorials/CNTK_302A_Evaluation_of_Pretrained_Super-resolution_Models_test.py

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+# Licensed under the MIT license. See LICENSE.md file in the project root
+# for full license information.
+# ==============================================================================
+
+import os
+import re
+import numpy as np
+
+abs_path = os.path.dirname(os.path.abspath(__file__))
+notebook = os.path.join(abs_path, "..", "..", "..", "..", "Tutorials", "CNTK_302A_Evaluation_of_Pretrained_Super-resolution_Models.ipynb")
+datadir = os.path.join(abs_path, "..", "..", "..", "..", "Examples", "Image", "DataSets", "BerkeleySegmentationDataset")
+
```

Comment:
Add the following:

# Run this on GPU only
notebook_deviceIdsToRun = [0]
notebook_timeoutSeconds = 1800

Reply:
notebook_deviceIdsToRun = [0]
notebook_timeoutSeconds = 600